exdb/tests/integration_tests.py

```diff
@@ -187,39 +194,53 @@ def setUp(self):
         self.test_org = self.create_org()
         self.test_keyword = self.create_keyword()
 
+    def get_post_data(self, start, end, name='test', description='test', ex_type=None, sub_type=None,
+                      guest='1', recognition=None, keywords=None, goal='a', action='submit'):
+
+        ex_type = ex_type or self.test_type.pk
+        sub_type = sub_type or self.test_sub_type.pk
+        recognition = recognition or [self.test_org.pk]
+        keywords = keywords or [self.test_keyword.pk]
```

Comment:
While these `or` clauses are nicer to look at than full-on if-statements, they make make it impossible to set falsy values such as `0` or `[]`.